### ISSUE TITLE
Can AliasMode RRs have parameters?

### ISSUE THREAD

**Issue by paulehoffman** [fl1ger/deleg#38]
The current text has no examples of priorty=0 with parameters, but it doesn't explicitly forbid that. Is there a reason to allow ipv4hint or ipv46hint with aliasing?

I'm not saying we should prohibit parameters for AliasMode, because there might be later parameters that could affect aliasing that we haven't thought of yet (or I could be wrong and there is a reason to have ipv4hint or ipv46hint with aliasing). But we should document what we know now.

**Comment by bemasc** [comment]
ipv4hint and ipv6hint are probably not relevant in AliasMode, but I do think we can imagine parameters that would be relevant.  For example, a "weight" parameter for unbalanced load spreading might be applicable to DELEG RRSets that contain multiple AliasMode records (or a mix of AliasMode and ServiceMode).

EDIT: Another example: a "dbound=0" SvcParam, indicating that this delegation is not an organizational boundary, could logically be applied to AliasMode DELEG records.

**Comment by pspacek** [comment]
I agree that parameters should be generally allowed. I can imagine that the first document which technically defines the mechanism will have the list of parameters empty and they will get utilized only by subsequent extensions.
